docs/sources/tempo/operations/prometheus_datasource.md

```diff
@@ -100,5 +100,4 @@ introduced with this API.  There is also a `rate()` function.
 
 Finally, the `by (resource.service.name)` groups the results into each series.
 
-<p align="center"><img src="explore.png" alt="Metrics visualization in Grafana "></p>
-![Caption](/media/docs/tempo/metrics-explore-sample.png)
+![Metrics visualization in Grafana](/media/docs/tempo/metrics-explore-sample.png)
```

Comment:
Nice.

If you want more control over the styling of the image, consider using the [`figure` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#figure)